wp-tevko-responsive-images.php

```diff
@@ -308,68 +307,86 @@ function tevkori_get_src_sizes( $id, $size = 'thumbnail' ) {
  * @param string $content The raw post content to be filtered.
  */
 function tevkori_filter_content_images( $content ) {
-	return preg_replace_callback(
-		'/<img\s([^>]+)>/i',
-		
-		// Don't use an anonymous callback function because it isn't supported by PHP 5.2.
-		'tevkori_filter_content_images_callback',
-		$content
-	);
+	// Only match images in our uploads directory
+	$uploads_dir = wp_upload_dir();
+	$path_to_upload_dir = $uploads_dir['baseurl'];
+
+	$content = preg_replace_callback( '|<img ([^>]+' . $path_to_upload_dir . '[^>]+) />|', '_tevkori_filter_content_images_callback', $content );
```

Comment:
```|<img ([^>]+' . $path_to_upload_dir . '[^>]+) />|```

The slash before the closing tag is optional in html5 so it would be good if the regex reflects that in case there are editors/page builders that omit it.
I think the space between ```)``` and ```/``` should be removed.
Might be good to add the ```i```modifier to make it case insensitive.